toolkit/tools/internal/srpm/srpm_test.go

```diff
@@ -0,0 +1,117 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+package srpm
+
+import (
+	"os"
+	"path/filepath"
+	"testing"
+
+	"github.com/microsoft/CBL-Mariner/toolkit/tools/internal/logger"
+	"github.com/stretchr/testify/assert"
+)
+
+func TestMain(m *testing.M) {
+	logger.InitStderrLog()
+	os.Exit(m.Run())
+}
+
+func TestShouldReturnArrayOfSpecs(t *testing.T) {
+	inputSet := []string{
+		"test-a",
+	}
+	var emptyInputSet []string
+	specDir := "./testout"
+	fileA := "test-a.spec"
+	fileB := "test-b.spec"
+	filepathA := filepath.Join(specDir, fileA)
+	filepathB := filepath.Join(specDir, fileB)
+
+	os.Mkdir(specDir, 0777)
+	defer os.Remove(specDir)
+
+	fileAPtr, err := os.Create(filepathA)
+	assert.NoError(t, err)
+	defer os.Remove(filepathA)
+	defer fileAPtr.Close()
+
+	fileBPtr, err := os.Create(filepathB)
+	assert.NoError(t, err)
+	defer os.Remove(filepathB)
+	defer fileBPtr.Close()
```

Comment:
For test files we tend to use just checked-in ones. This way we can also reduce the amount of boilerplate and clean-up code in the tests and focus better on the tested scenario. If you don't mind, could we convert this set-up into 2 dummy spec files?